<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Square">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FrequencyButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputStartFreq">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Sweep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sweep</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputEndFreq">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End  Frequency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setVoltageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Amplitude</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputVoltage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFreq">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSquareFreq">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RampButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputRampFreq">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSinFreq">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pulseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPulseFreq">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>51</width>
      <height>291</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wave Function</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Amplitude Vpp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="offsetButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="phaseButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="offsetInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phaseInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Sin</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SinButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ConnectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Square</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SquareButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FrequencyButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>FrequencyButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sweep</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SweepButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sweep</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SweepButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ConnectButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DeviceStatusButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setVoltageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setVoltageButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RampButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>RampButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pulseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PulseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OffsetButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phaseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PhaseButtongClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phaseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PhaseButtongClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phaseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PhaseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SinButtonClicked()</slot>
  <slot>FrequencyButtonClicked()</slot>
  <slot>AmplitudeButtonClicked()</slot>
  <slot>ConnectButtonClicked()</slot>
  <slot>VoltageButtonClicked()</slot>
  <slot>SquareButtonClicked()</slot>
  <slot>SweepButtonPressed()</slot>
  <slot>SweepButtonClicked()</slot>
  <slot>DeviceStatusButton()</slot>
  <slot>setVoltageButton()</slot>
  <slot>RampButtonClicked()</slot>
  <slot>PulseButtonClicked()</slot>
  <slot>OffsetButtonClicked()</slot>
  <slot>PhaseButtonClicked()</slot>
 </slots>
</ui>
